code/modules/awaymissions/mission_code/centcomAway.dm

```diff
@@ -40,11 +40,11 @@
 	name = "Visitor Info Pamphlet"
 	info = "<b> XCC-P5831 Visitor Information </b><br>\
 	Greetings, visitor, to  XCC-P5831! As you may know, this outpost was once \
-	used as Nanotrasen's CENTRAL COMMAND STATION, organizing and coordinating company \
+	used as the ICC's central hub, organizing and coordinating corporate \
 	projects across the vastness of space. <br>\
 	Since the completion of the much more efficient CC-A5831 on March 8, 2553, XCC-P5831 no longer \

```

Comment:
We still keeping the same year?

Reply:
I haven't decided on year yet, but probably not.